Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/06-SyncUpDetail/TestingSyncUpDetail-02-code-0005.swift

```diff
@@ -11,17 +11,21 @@ struct SyncUpDetailTests {
       id: SyncUp.ID(),
       title: "Point-Free Morning Sync"
     )
+    
+    @Shared(.syncUps) var syncUps = [syncUp]
+
     let store = TestStore(initialState: SyncUpDetail.State(syncUp: Shared(value: syncUp))) {
       SyncUpDetail()
     }
 
     await store.send(.deleteButtonTapped) {
       $0.destination = .alert(.deleteSyncUp)
     }
-    await store.send(.destination(.presented(.alert(.confirmButtonTapped)))) {
+    await store.send(\.destination.alert.confirmButtonTapped) {
       $0.destination = nil
     }
-    await store.receive(\.delegate.deleteSyncUp, syncUp.id)
+  
+    assertNoDifference(syncUps, [])
```

Comment:
📓 This new step explain how to test that the list of sync-ups has been updated by then checking manually the update of the list.